api-contracts/openapi/paths/tenant/tenant.yaml

```
@@ -468,3 +468,55 @@ members:
     summary: List tenant members
     tags:
       - Tenant
+member:
+  delete:
+    x-resources: ["tenant", "member"]
```

```suggestion
    x-resources: ["tenant"]
```

The `x-resources` tag is used to populate resources into the context automatically. All of these "getters" are registered starting here: https://github.com/hatchet-dev/hatchet/blob/31d11c93a29a6eb3aeb2fd68b4ebd84acc436734/api/v1/server/run/run.go#L86. 

So there's no need for this unless I'm misunderstanding.